### DISCUSSION TITLE
请问 umi 如何使用 vite 的 import.meta.glob

### DISCUSSION THREAD

**Discussion by 0x30** [umijs/umi#11791] [Q&A]
请问 umi 如何使用 vite 的 import.meta.glob

**Comment by fz6m** [comment]
开启 `vite: {}` 模式后就会使用 vite 4 运行项目了，直接用就行了。

**Comment by liqiushi720** [comment]
为什么我开启vite: {} 模式后会直接报这个错误Error: Can't resolve '@/types'

**Reply by fz6m** [reply]
目前 vite 模式为实验性功能，用于开发 vue 或单纯的项目，不支持同时开启 umi 的其他插件功能，如有问题，需要给一个最小复现。
